agent/doc/UML/test_runner_design.puml

```diff
@@ -114,7 +114,7 @@ deactivate TaskCompletion
 }
 @endjson
 
-@startuml design
+@startuml TestRunnerListenerDesign
```

Comment:
Use underlying pattern to align to the other UMLs' naming.

Reply:
```suggestion
@startuml test_runner_listener_design
```